components/Gallery.tsx

```diff
@@ -0,0 +1,67 @@
+import { LinearGradient } from "expo-linear-gradient";
+import React from "react";
+import { Image, StyleSheet } from "react-native";
+import PagerView from "react-native-pager-view";
+import { View, Text } from "./Themed";
+
+export default function Gallery() {
+  return (
+    <View style={styles.container}>
+      <PagerView style={styles.pagerView} initialPage={0} showPageIndicator={true}>
+        <View key="1" style={styles.page}>
```

Comment:
it's probably not good to hard code the number of views. the user can add any number of images as they wish (up to a limit). there's also lots of code duplication here